app/components/stepper.hbs

```diff
@@ -27,66 +27,77 @@
       </h3>  
     {{/if}}
   {{/if}}
-<form class="stepper__form" onsubmit={{this.nextStep}}>
-  {{! Content }}
-  {{#if (eq this.currentStep 1)}}
-    <JoinSteps::StepOne @setIsPreValid={{this.setIsPreValid}} @isValid={{this.isValid}} @setIsValid={{this.setIsValid}} />
-  {{else if (eq this.currentStep 2)}}
-    <JoinSteps::StepTwo @setIsPreValid={{this.setIsPreValid}} @isValid={{this.isValid}} @setIsValid={{this.setIsValid}}/>
-  {{else if (eq this.currentStep 3)}}
-    <JoinSteps::StepThree @setIsPreValid={{this.setIsPreValid}} @isValid={{this.isValid}} @setIsValid={{this.setIsValid}}/>
-  {{else if (eq this.currentStep 4)}}
-    <JoinSteps::StepFour login={{this.login}} />
-  {{else if (eq this.currentStep 5)}}
-    <JoinSteps::StepFive />
-  {{/if}}
+  {{#if (and (eq this.currentStep 0) (not this.login.isLoggedIn))}}
```

Comment:
Don intrupt the current stepper.hbs file
I created stepper-signup.hbs component- 
https://github.com/Real-Dev-Squad/website-www/pull/607/files#diff-7bad8c77216aba9edbc363d2c213847ebf39040a04b5c9ade78419fca26b8584R1

@rohan09-raj suggested above idea